# Goal Definition
std_msgs/Header header
geometry_msgs/Pose2D desired_pose
string which_arm
string controller_name
string proxy_name
string behavior_primitive
---
# Result Definition
std_msgs/Header header
geometry_msgs/Pose2D x
geometry_msgs/Pose2D x_error
bool aborted
---
# Feedback Definition
std_msgs/Header header
geometry_msgs/Pose2D x
geometry_msgs/Pose2D x_dot
float64 z
bool no_detection
string controller_name
string proxy_name
string behavior_primitive